content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Running Open 3D Multiplayer Projects
+description: How to run Multiplayer enabled projects and relevant configuration
+linkTitle: Running Multiplayer
+---
+
+Running projects using the Open 3D Engine Multiplayer Gem requires build steps followed by a variety of options to run the projects.
+
+* Ensure your project is [properly configured](./configuration.md)
```

Comment:
Does this cover how to add and build the Multiplayer gem to you project?

Reply:
Yep